source/air/src/typecheck.rs

```diff
@@ -89,6 +89,7 @@ fn check_typs(typing: &Typing, typs: &[Typ]) -> Result<(), TypeError> {
 }
 
 fn check_exprs(
```

Comment:
Could we add a copy of `message_interface` to `Typing` to avoid passing an extra argument?